lib/sync.js

```diff
@@ -85,39 +93,38 @@ const update = (options) => {
     return acc;
   }, {});
 
-  debug('found %d dictionaries, attempting to update modules', Object.keys(files).length);
+  debug('found %d module ids, attempting to update module files', Object.keys(moduleUpdates).length);
 
-  Object.keys(files).forEach((id) => {
-    const filePath = path.resolve(reverseModuleId(id, options.keysRoot, options.moduleEnding));
-    const content = JSON.stringify(files[id], null, 2);
-    debug('saved %s module to %s', id, filePath);
+  Object.keys(moduleUpdates).forEach((id) => {
+    const filePath = moduleFiles[id];
+    if (!filePath) return debug('module file for %s doesn\'t exist', id);
```

Comment:
```suggestion
    if (!filePath) { 
      debug('module file for %s doesn\'t exist', id);
      return;
    }
```

Reply:
i will pad with spaces a bit, i see no reason to split into multiple lines

Reply:
My point is to not return values that aren't used

Reply:
i agree in general, but within a forEach it doesn't matter